lib/__tests__/fixtures/rule-empty-line-before.html

```diff
@@ -5,6 +5,9 @@
 h1 {
```

Comment:
Would the following?

```css
@media (min-width: 100px) {
  h1 {
    font-weight: normal;
  }
}
```

And no warnings be a more appropriate test for this change as your fixing false positives, right?

Reply:
> Would the following?
It has nothing to do with this.
The affected is that the <style> tag first-level child element is misjudged as nested.

> And no warnings be a more appropriate test for this change as your fixing false positives, right?

It is not "false positives", It is "missed warnings"
(I don't know if my English is appropriate, title modified, please help correct English grammar)

Reply:
> The affected is that the <style> tag first-level child element is misjudged as nested.

Oooo, I see now. Sorry, I misunderstood. 

> It is not "false positives", It is "missed warnings"

I've just been using "false negatives" for those :)